tests/performance/load/groups/mainDashboard.js

```diff
@@ -5,37 +5,53 @@ export function mainDashboard(host, headers, tags) {
     group('main dashboard', function () {
         http.post(
             `${host}/api/graphql?opname=summary_counts`,
-            '{"operationName":"summary_counts","variables":{},"query":"query summary_counts {\\n  clusterCount\\n  nodeCount\\n  violationCount\\n  deploymentCount\\n  imageCount\\n  secretCount\\n}\\n"}',
+            '{"operationName":"summary_counts","variables":{},"query":"query summary_counts {\\n  clusterCount\\n  nodeCount\\n  violationCount\\n  deploymentCount\\n  imageCount\\n  secretCount\\n}"}',
```

Comment:
nit: how about making it more redable:
```js
JSON.stringify({
                "operationName": "summary_counts",
                "variables": {},
                "query": `query summary_counts {
                    clusterCount
                    nodeCount
                    violationCount
                    deploymentCount
                    imageCount
                    secretCount
                }`
            })
```

Reply:
#14191 
I'll have to compare roughly the http_req_wait values between the state of this PR and the state of the follow-up one.